FBC/FunctionalEntities/BusinessCenters.rdf

```diff
@@ -92,7 +93,7 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>business day adjustment code</rdfs:label>
-		<skos:definition>a code used to denote convention for specifying what happens when a date falls on a day that is weekend or holiday in some municipality or business center</skos:definition>
+		<skos:definition>code used to denote a convention for specifying what happens when a date falls on a day that is weekend or holiday in some municipality or business center</skos:definition>
```

Comment:
should not say "some municipality" but "the municipality relevant to the contract or transaction"